checkov/common/checks_infra/solvers/resource_solvers/base_resource_solver.py

```diff
@@ -49,6 +49,8 @@ def run(
 
         for _, data in graph_connector.nodes():
             result = self.get_operation(resource_type=data.get(CustomAttributes.RESOURCE_TYPE))
+            if result is None:
```

Comment:
seems like we have `unknown_vertices` so it should probably be appended there

Reply:
After clarifying things with @maxamel, I appended the None results to unknown also here.